docs/streams/developer-guide/config-streams.html

```diff
@@ -216,7 +216,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
 //streamsSettings.put(StreamsConfig.REPLICATION_FACTOR_CONFIG, 3);
 streamsSettings.put(StreamsConfig.topicPrefix(TopicConfig.MIN_IN_SYNC_REPLICAS_CONFIG), 2);
 streamsSettings.put(StreamsConfig.producerPrefix(ProducerConfig.ACKS_CONFIG), "all");
```

Comment:
Could you please fix line#217 and line#218 also?

Reply:
well, I think line#217 and line#218 are legit